mlflow/utils/databricks_utils.py

```diff
@@ -335,6 +335,38 @@ def get_workspace_url():
         return None
 
 
+def is_mlflowdbfs_available():
+    if not is_in_cluster():
+        return False
+
+    # allow users to choose not to use mlflowdbfs via an environment variable
+    disable_mlflowdbfs = os.environ.get("DISABLE_MLFLOWDBFS")
+    if disable_mlflowdbfs is not None and disable_mlflowdbfs != "":
+        return False
+
+    try:
+        spark_session = _get_active_spark_session()
+    except Exception:
+        return False
+
+    if spark_session is None:
+        return False
+
+    from py4j.protocol import Py4JJavaError
+
+    try:
+        spark_session.read.load("mlflowdbfs:///123")
+    except Py4JJavaError as e:
+        if (
+            str(e.java_exception)
+            == "org.apache.hadoop.fs.UnsupportedFileSystemException:"
+            + ' No FileSystem for scheme "mlflowdbfs"'
```

Comment:
Comparing against this exact message seems brittle. Perhaps we want to check that the message contains `No FileSystem for scheme "mlflowdbfs"`?